.rubocop.yml

```diff
@@ -1,13 +1,14 @@
 inherit_from: .rubocop_todo.yml
 
-require:
+plugins:
   - rubocop-i18n
   - rubocop-performance
   - rubocop-rake
   - rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 3.1
+  TargetRubyVersion: 3.2
```

Comment:
If we do this, we would need to bump `required_ruby_version` in the gemspec as well. I never understood why Perforce decided to set ruby 3.1 as lower version but vendor Ruby 3.2 in the AIO.

Reply:
Hmm, yeah, I suppose we should.

Reply:
Ah, I think it's because JRuby 9.4 only supports 3.1.  Which makes it extra silly 3.2 is in the AIO.  JRuby 10 supports 3.4, but that's a lot more work to get there.  I should probably remove the `set` commit since that might not work with JRuby.

Reply:
Ah maybe that's why it's pinned to 3.1 :( IMO we should stick to that version until we do a major release.